src/three-components/TextureUtils.ts

```diff
@@ -78,10 +78,13 @@ export default class TextureUtils extends EventDispatcher {
   private[$environmentMapCache] = new Map<string, Promise<WebGLRenderTarget>>();
   private[$skyboxCache] = new Map<string, Promise<WebGLRenderTargetCube>>();
 
+  private[$PMREMGenerator]: PMREMGenerator;
```

Comment:
Lowercase for the first term in a "camel-case" identifier. So, `$pmremGenerator`.